index.html

```diff
@@ -188,6 +188,12 @@ <h2 class="cont-para">I'm always interested in hearing about new projects, so if
       </ul>
     </div>
   </footer>
+
+  <div class="project-popup">
+  </div>
+
+  <div id="overlay">
+  </div>
```

Comment:
- [ ] Just as the previous reviewer pointed out, the issue still persists. You will notice that the popup is not centered from the bottom part. Kindly center the pop-up to make it look like the Figma design as posted by the previous reviewer.
![image](https://user-images.githubusercontent.com/86200356/202432357-4dfd1614-f82b-48e4-affc-97f0c480e642.png)